src/main/scala/cromwell/Main.scala

```diff
@@ -18,11 +20,23 @@ object Actions extends Enumeration {
 
 object Main extends App {
 
+  val props = System.getProperties
+  val loggerProperty = "CROMWELL_LOGGER"
+  Option(props.getProperty(loggerProperty)) match {
```

Comment:
I'd prefer to keep this as tech debt since it was part of the logging PR and not my changes.

Also I'm not sure I agree. `getOrElse` is intended to return a value which here would be ignored; this code is run purely for its side-effect of setting the logger property.  :smile: